src/includes/beta-note-session-replay.mdx

```diff
@@ -1,5 +1,16 @@
 <Note>
 
-Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please open a [GitHub issue](https://github.com/getsentry/sentry/issues/new/choose). Alternatively, you can email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+
+**Currently, this feature is supported for only browser JavaScript, including these frameworks:**
+
+- Angular
```

Comment:
do we need to list these frameworks explicitly? as we'll work on other frameworks, there's just not a specific sentry sdk for them.

Reply:
We have this list on our SDK docs on our product docs - https://docs.sentry.io/product/session-replay/. Considering the volume of inquiries around if we formally support Electron, iOS, React Native, etc. I thought it might be helpful to repeat it here since these docs are viewed often

Maybe we could shorten this to something like - "Session Replay supports all browser JavaScript applications. It is built to work with @sentry/browser, and our browser framework SDKs." instead of individually listing? What do you think?

Reply:
sounds good to me :) may be worth mentioning that it _doesn't_ currently support "Electron, iOS, React Native"